scripts/storage-emulator-integration/multiple-targets/tests.ts

```diff
@@ -0,0 +1,65 @@
+import supertest = require("supertest");
+import { TriggerEndToEndTest } from "../../integration-helpers/framework";
+import {
+  EMULATORS_SHUTDOWN_DELAY_MS,
+  FIREBASE_EMULATOR_CONFIG,
+  getStorageEmulatorHost,
+  readEmulatorConfig,
+  TEST_SETUP_TIMEOUT,
+} from "../utils";
+
+const FIREBASE_PROJECT = process.env.FBTOOLS_TARGET_PROJECT || "fake-project-id";
+
+describe("Multiple Storage Deploy Targets", () => {
+  let test: TriggerEndToEndTest;
+  const storageBucket = `${FIREBASE_PROJECT}.appspot.com`;
+  const otherStorageBucket = `${FIREBASE_PROJECT}-2.appspot.com`;
+  const emulatorConfig = readEmulatorConfig(FIREBASE_EMULATOR_CONFIG);
+  const STORAGE_EMULATOR_HOST = getStorageEmulatorHost(emulatorConfig);
+
+  before(async function (this) {
+    this.timeout(TEST_SETUP_TIMEOUT);
+
+    test = new TriggerEndToEndTest(FIREBASE_PROJECT, __dirname, emulatorConfig);
+    await test.applyTargets(["storage", "main", storageBucket]);
+    await test.applyTargets(["storage", "other", otherStorageBucket]);
+    await test.startEmulators(["--only", "storage"]);
```

Comment:
do we not want the auth emulator?

Reply:
Nope, no need to authenticate any requests. Either the rules allow all or allow none.